### ISSUE TITLE
git Repo integration

### ISSUE THREAD

**Issue by mpldr** [mirego/accent#446]
Hi, I would like to propose git integration in a similar fashion as Weblate has it. So this would include:

- Syncing translation files straight from the repos (this is probably useful for anyone)
- Translations as Commits with proper author attribution (would be nice to the volunteers and better for blames)
  - Bonus points for properly amending commits, so every user is only creating one commit at a time
- Option to either push to source or exporting the commits (nice for patch-based workflows)

**Comment by AerunDev** [comment]
Yeah I'm using Weblate for another project and that could be awesome!
